packages/@okta/vuepress-site/docs/release-notes/2021/index.md

```diff
@@ -1,6 +1,52 @@
+| [Support for Push Status using the Apps API is GA in Production](#support-for-push-status-using-the-apps-api-is-ga-in-production)| August 4, 2021 |
+| [The Device Authorization grant type is Self-Service EA](#the-device-authorization-grant-type-is-self-service-ea) | August 4, 2021 |
+| [Token-based SSO between native apps is Self-Service EA](#token-based-sso-between-native-apps-is-self-service-ea) | August 4, 2021 |
+
+#### Okta Sign-In Widget, version 5.9.0
+
+For details about this release, see the Okta [Sign-In Widget Release Notes](https://github.com/okta/okta-signin-widget/releases/tag/okta-signin-widget-5.9.0). For more information about the Widget, see the Okta [Sign-In Widget Guide](/code/javascript/okta_sign-in_widget/). <!--OKTA-416419-->
+
+#### Brands API is Self-Service EA
+
+The Okta [Brands API](/docs/reference/api/brands/) allows customization of the look and feel of pages and templates. It allows you to upload your own brand assets (colors, background image, logo, and favicon) to replace Okta's default brand assets. You can then publish these assets directly to the Okta-hosted Sign-In Page, error pages, email templates, and the Okta End-User Dashboard. See [Customize your Okta experience with the Brands API](/docs/guides/customize-themes/). <!-- 414350-->
+
+#### Risk Providers API is EA
+
+The Okta Risk Providers API enables security teams to integrate IP-based risk signals to analyze and orchestrate risk-based access using the authentication layer. Practitioners can step up, reduce friction, or block the user based on risk signals across the customer's security stack. Apart from improving security efficacy, this feature also enhances the user experience by reducing friction for good users based on positive user signals. See [Third-party risk provider integration](/docs/guides/third-party-risk-integration/overview/). <!-- OKTA-415574 -->
+
+#### SAML parameter SessionNotOnOrAfter is GA in Preview
+
+The SAML parameter `SessionNotOnOrAfter` for SAML assertions is available for use with Okta's [SAML Inline Hooks](/docs/reference/saml-hook/). This optional parameter specifies the session lifetime, in seconds, and is included in the SAML assertion. The `SessionNotOnOrAfter` parameter allows the Identity Provider to control the session of the Service Provider. Most SAML applications manage their own sessions. However, some SAML applications require this parameter from the Identity Provider for session management. <!--OKTA-406633-->
+
+#### Support for Push Status using the Apps API is GA in Production
+
+Developers can use the [`pushStatus` parameter](/docs/reference/api/apps/#username-template-object) to handle a username update to an app integration. Previously, this option wasn't available through the Apps API, which caused inconsistent behavior between app integrations configured using the Okta Admin Console and those configured through the API. <!--OKTA-413817-->
```

Comment:
To avoid the ugly styling, I suggest having the link only belong to "parameter". As it is, it looks like two links anyway.